pwndbg/gdblib/disasm/arm.py

```diff
@@ -117,6 +146,17 @@ def _set_annotation_string(self, instruction: PwndbgInstruction, emu: Emulator)
                 ARM_SINGLE_STORE_INSTRUCTIONS[instruction.id],
                 instruction.operands[1].str,
             )
+        elif instruction.id in ARM_EXCLUSIVE_STORE_INSTRUCTIONS:
```

Comment:
Btw shall we do sth like:
```
if (ins_id := instruction.id) in ARM_SINGLE_LOAD_INSTRUCTIONS: ...
elif ins_id in ...: ...
# etc
```
?

Also, that's probably wishful thinking but it would be nice to order those if/elifs via the frequency in which those instructions appear in disassembly :D